### PR TITLE
new(kernel_crawler): rename distro keys to match os-release names.

### PR DESCRIPTION
**Pull request by FedeDP**
This is what we expect in driverkit, dbg-go and test-infra.

**This is a BREAKING CHANGE**

/cc @maxgio92 @EXONER4TED
/hold for discussion.

See previous #185

### CODE REVIEW

**Comment by EXONER4TED** [comment]
I'm all for this, been wondering why they weren't synced up haha. I am not in office today though - I have a sick kiddo I'm taking care of 🙁 

I'll approve this tomorrow when I get in so that I can make the necessary changes downstream to handle it.

Thanks!

**Comment by FedeDP** [comment]
> I have a sick kiddo I'm taking care of 🙁

Welcome to new school's year i guess? :rofl: 

> I'll approve this tomorrow when I get in so that I can make the necessary changes downstream to handle it.

Great!

**Comment by FedeDP** [comment]
Also @EXONER4TED please note that, as stated here: https://github.com/falcosecurity/test-infra/issues/1118#issuecomment-1775280917 i force pushed main branch of kernel-crawler removing any fat json from the history (since at the start of the repo life, kernels lists were pushed to the main branch, not the `kernels` one).
This was made to reduce repo size and hopefully fix https://github.com/falcosecurity/test-infra/issues/1118 issue.
I think we must cleanup something else too, ie: there are still unreferenced blobs objects in the repo.

File: .github/workflows/update-kernels.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 name: Update Kernels
 
 on:
+  workflow_dispatch:
```
Comment (FedeDP) [inline .github/workflows/update-kernels.yml:5]: This is needed to trigger a refresh of kernels lists, allowing https://github.com/falcosecurity/dbg-go/pull/36 CI to be green.

Comment (FedeDP) [reply .github/workflows/update-kernels.yml:5]: Anyway, we can leave it there, it makes no harm after all.

**Review by EXONER4TED — APPROVED** [review]
Back in office today - approving

**Pull request merged**